prime-dotnet-webapi/Services/EmailRenderingService.cs

```diff
@@ -229,7 +229,7 @@ public async Task<Email> RenderEnrolleeAbsenceNotificationEmailAsync(string emai
             (
                 from: PrimeEmail,
                 to: email,
-                subject: "Lorem ipsum",
+                subject: "PRIME Absence Notification",
```

Comment:
I can't comment on the config file, but the text in the ticket has "This is an automated generated email from PRIME." not "To Whom it may concern,"